kinematics.go

```diff
@@ -61,10 +47,9 @@ type XyzWxyz struct {
 
 // ForwardKinematics calculates the end effector XyzWxyz coordinates given
 // joint angles and robotic arm parameters.
-func ForwardKinematics(thetas StepperTheta, dhParameters DhParameters) XyzWxyz {
+func ForwardKinematics(thetas []float64, dhParameters DhParameters) XyzWxyz {
 	// First, setup variables. We use 4 variables - theta, alpha, a and d to calculate a matrix
 	// which is then multiplied to an accumulator matrix.
-	thetaArray := []float64{thetas.J1, thetas.J2, thetas.J3, thetas.J4, thetas.J5, thetas.J6}
 	var theta float64
 	var alpha float64
 	var a float64
```

Comment:
Should we make an identity matrix creating function? This is generally useful I've noticed. Maybe we could make some "utils" file for those kind of things?

Reply:
Not necessary I don't think, since it's only used in one place.

Reply:
Ie, generally useful would go in a different package somewhere else. We just do kinematics, nothin' else.